### DISCUSSION TITLE
navigator.lua

### DISCUSSION THREAD

**Discussion by realtica** [mfussenegger/nvim-jdtls#457] [Q&A]
How is the correct way to configure nvim-jdtls to work together with [navigator.lua](https://github.com/ray-x/navigator.lua)??

**Comment by ZakharE** [comment]
@realtica hi! Did you figure it out? Could you share your config, please?

**Comment by EpsilonKu** [comment]
@ZakharE it should work from box. Can u share ur config?

**Reply by ZakharE** [reply]
Here is nvim-jdtls setup  https://github.com/ZakharE/nvim/blob/main/ftplugin/java.lua 
When i try to use lsp hover feature on `println` method it says that "no manual entry found for println"
<img width="946" alt="image" src="https://github.com/mfussenegger/nvim-jdtls/assets/28906224/14c13d78-4379-49b0-bfdf-eafbd06e1a8d">

**Reply by EpsilonKu** [reply]
You need configure decompiler like `fernflower`.

**Reply by ZakharE** [reply]
I've found a problem. I forgot to add 
```
 require('navigator.lspclient.mapping').setup({client=client, bufnr=bufnr}) -- setup navigator keymaps here,

      require("navigator.dochighlight").documentHighlight(bufnr)
      require('navigator.codeAction').code_action_prompt(bufnr)
```
from this [section](https://github.com/ray-x/navigator.lua#integration-with-other-lsp-plugins-eg-rust-tools-gonvim-clangd-extension) to my java.lua file 😞  Now it works perfectly
